app/models/ability.rb

```diff
@@ -5,6 +5,7 @@ def initialize(user)
     user ||= User.new
 
     # Abilities that everyone get.
+    can :index, Tool
```

Comment:
Ah. I don't necessarily see the point in splitting the application logic in this way but then the patch makes sense.